ServerDevcommands/Commands/Dmg.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ServerDevcommands;
+public class DmgCommand
+{
+    public DmgCommand()
+    {
+        Helper.Command("dmg", "[target,,value] - Default value: 5. (Negative values heal the character).", (args) =>
```

Comment:
Server Devcommands supports using " in commands.

For example `dmg "player name" 5` would work.

In this case you can also use the last arg as damage and then join other args with `string.Join(" ", ...);`

Single , as separator is also ok but I think double looks bit weird.